README.md

```diff
@@ -39,60 +40,62 @@ conda install -c conda-forge woodwork
 
 ## Example
 
-Below is an example of using Woodwork. In this example, a sample dataset of order items is used to create a Woodwork `DataTable`, specifying the `LogicalType` for three of the columns.
+Below is an example of using Woodwork. In this example, a sample dataset of order items is used to create a Woodwork `DataFrame`, specifying the `LogicalType` for three of the columns.
 
 ```python
 import woodwork as ww
 
-data = ww.demo.load_retail(nrows=100, return_dataframe=True)
-dt = ww.DataTable(data, name='retail')
-dt.set_types(logical_types={
+df = ww.demo.load_retail(nrows=100, return_dataframe=True)
+df.ww.init(name='retail')
+df.ww.set_types(logical_types={
     'quantity': 'Double',
     'customer_name': 'Categorical',
     'country': 'Categorical'
```

Comment:
Should we add `order_id` to be `Categorical`?

Reply:
Yes, that makes sense. Changed.